flake.nix

```diff
@@ -460,6 +460,35 @@
               cargoBuildCommand = "cargo build --release -p price-feed";
             });
 
+            fmt = pkgs.writeShellApplication {
+              name = "fmt-composable";
+
+              runtimeInputs = with pkgs; [
+                nixfmt
+                coreutils
+                rust-nightly
+                taplo
+                nodePackages.prettier
+              ];
+
+              text = ''
+                  # .nix 
+                	find . -name "*.nix" -type f -exec nixfmt {} \;
```

Comment:
nix checked else where afaik. and used xarga i stead of exec. as xargs more composable and no need detals of adhoc exec

Reply:
Oh yeah good point, will convert to xargs

Reply:
fixed